pkg/operator/context/transformers.go

```diff
@@ -81,6 +82,7 @@ func newTransformer(
 	transConfig userconfig.Transformer,
 	impl []byte,
 	namespace *string,
+	pythonPackages *context.PythonPackages,
```

Comment:
This also doesn't have to be a pointer, and the `if pythonPackages != nil` below can be removed